ode-wrapper/src/main/java/com/trihydro/odewrapper/helpers/SetItisCodes.java

```diff
@@ -32,13 +32,17 @@ public class SetItisCodes {
 
     private List<ItisCode> itisCodes;
 
+    private final String AdvisoryItisCode = "7712";
+    private final String ReducedItisCode = "12302";
+    private final String AheadItisCode = "13569";

```

Comment:
Suggestion (nitpick): Constants like this should be in uppercase format like ADVISORY_ITIS_CODE (see https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)